### DISCUSSION TITLE
Docstring in Intellisense Quick Info looks messy

### DISCUSSION THREAD

**Discussion by forallsunday** [microsoft/pylance-release#2655] [Enhancement]
Two screens shots of Pylance and Jedi, the quick info of intellisense.

![pylance1](https://user-images.githubusercontent.com/41034184/145659769-bb290c51-fdfa-4d7e-ba5b-2698e370156d.png)
![jedi2](https://user-images.githubusercontent.com/41034184/145659771-fa4e3dc0-e044-454e-810e-19e002d72282.png)

Here's two way I think pylance should be improved:

1.  The function signature shows in one line, it doesn't warp, so I have to drag the horizontal scroll bar by mouse. Inconveniently.
2.  Parameters are not in "code blocks", and not in a list view. Looks disordered.

Compared with Pylance, Jedi looks much better, the doc string looks more clearly.
Please improve the quick info.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
